typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -361,7 +374,8 @@ export abstract class HyperlaneAppGovernor<
     }
 
     // 2. Check if the call will succeed via Gnosis Safe.
-    const safeAddress = this.checker.configMap[chain].owner;
+    const safeAddress =
+      this.checker.configMap[chain].ownerOverrides?.safeAddress;
```

Comment:
one edge case I could see us hitting is that there may not always be a single Safe address on a given chain

We already have this today to some degree (we have some "old Safes" you can see in Notion that still own some contracts, though they generally own v2)

But I think in the not so far future we'll have multiple safes -- e.g. how there's gonna be a separate Safe that will own the Proxy Admin. In this situation we'll have this new Safe set as the `proxyAdmin` owner in the ownerOverrides

So wondering if we should check this plumbed in safeAddress in addition to checking `configMap[chain].owner;` instead of in lieu

Reply:
answered in a diff comment